crates/oxc_language_server/src/main.rs

```diff
@@ -353,16 +353,21 @@ impl Backend {
         }
         if let Some(config_path) = config_path {
             let mut linter = self.server_linter.write().await;
+            let oxlintrc = Oxlintrc::from_file(&config_path)
+                .expect("should have initialized linter with new options");
+            let mut gitignore_builder = GitignoreBuilder::new(config_path.parent().unwrap());
+            for path in &oxlintrc.ignores {
+                gitignore_builder
+                    .add_line(None, path)
+                    .expect("Invalid glob in config \"ignores\" property");
+            }
+            self.gitignore_glob.lock().await.push(gitignore_builder.build().unwrap());
```

Comment:
This modifies `self.gitignore_glob` after it has been created in `init_ignore_glob`.  This might be confusing and things could probably be refactored so that it isn't modified outside of `init_ignore_glob`.